systemtest/src/test/java/io/strimzi/systemtest/operators/LeaderElectionIsolatedST.java

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.operators;
+
+import io.fabric8.kubernetes.api.model.EnvVar;
+import io.fabric8.kubernetes.api.model.EnvVarBuilder;
+import io.fabric8.kubernetes.api.model.PodBuilder;
+import io.fabric8.kubernetes.api.model.coordination.v1.Lease;
+import io.strimzi.systemtest.AbstractST;
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.Environment;
+import io.strimzi.systemtest.annotations.IsolatedSuite;
+import io.strimzi.systemtest.storage.TestStorage;
+import io.strimzi.systemtest.utils.StUtils;
+import io.strimzi.systemtest.utils.kubeUtils.objects.PodUtils;
+import io.strimzi.test.annotations.IsolatedTest;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.extension.ExtensionContext;
+
+import java.util.Arrays;
+import java.util.List;
+
+import static io.strimzi.systemtest.Constants.INFRA_NAMESPACE;
+import static io.strimzi.systemtest.Constants.REGRESSION;
+import static io.strimzi.systemtest.resources.ResourceManager.kubeClient;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+
+/**
+ * Suite for testing Leader Election feature <br>
+ *
+ * The feature allows users to run Cluster operator in more than one replica <br>
+ *
+ * There will be always one leader, other replicas will stay in "standby" mode <br>
+ *
+ * The whole procedure of deploying CO with Leader Election enabled and many more is described in
+ *
+ * <a href="https://strimzi.io/docs/operators/in-development/configuring.html#assembly-using-multiple-cluster-operator-replicas-str">the documentation</a>
+ */
+
+@Tag(REGRESSION)
+@IsolatedSuite
+public class LeaderElectionIsolatedST extends AbstractST {
+
+    private static final Logger LOGGER = LogManager.getLogger(LeaderElectionIsolatedST.class);
+
+    /*
+     These env variables are already inside packaging/../060-Deployment-strimzi-cluster-operator.yaml, but to be sure that
+     we are setting everything correctly and the Leader Election is really enabled, we are adding it to additional envs
```

Comment:
> These env variables are already inside packaging/../060-Deployment-strimzi-cluster-operator.yaml, but to be sure that we are setting everything correctly and the Leader Election is really enabled, we are adding it to additional envs

well, I see your point about overriding these envs. But we relay on `packaging/../060-Deployment-strimzi-cluster-operator.yaml` file. Meaning that Leader Election is enabled by default (with 1 replica) and thus we expect that such Deployment will have these ENVs already correctly configured. By overriding, we could simply miss a real bug. Maybe we can easily do a pre-check (i.e., check if such ENVs are present in `060-Deployment-strimzi-cluster-operator.yaml`? If so we continue, otherwise throw an Exception?

Reply:
I agree with @see-quick. There is no plan to disable it in the 060-... file. And by using what it has, you don't test only the code but also the installation files which adds more value.

Reply:
Okay, thanks both :)